integration/side-effects/snapshots/esm/ajax.js

```diff
@@ -1 +1 @@
-
+import 'tslib';
```

Comment:
It seems like using `URLSearchParams` pulls in `tslib` here. I was unaware that `tslib` polyfilled that. It might be the case that this "just works" in IE, which is weird, and almost unfortunate, because I'd rather not depend on that and not have it bundled in with this (to keep it smaller).

I guess this is something to discuss at the core team meeting then.

@cartant @kwonoj  do you any thoughts on this?

Reply:
(this comment isn't outdated, I just had to update the file after prettier mangled it and ruined the test)

Reply:
IDK anything about the tool that's generating this side-effects business, but my guess would be that it has more to do with the object rest destructuring that was added here:

https://github.com/ReactiveX/rxjs/blob/fa9415e9c25d3e7819c238b5d6da5e825434811b/src/internal/ajax/ajax.ts#L281

Reply:
Ah. That makes sense.